### PR TITLE
docs(linux): AM62AX: Fix installer name and MCU+SDK link

### PR DESCRIPTION
**Pull request by sinha-shreyash**

### CODE REVIEW

**Review by jeevantelukula — DISMISSED** [review]

File: source/devices/AM62AX/linux/Overview/Download_and_Install_the_SDK.rst [hunk 1]
```diff
@@ -97,8 +97,8 @@ Refer :ref:`Release Notes <Release-note-label>` for complete feature-set and fur
 .. note::
    Processor SDK Linux AM62A contains only the Linux specific source and application intended
```
Comment (cshilwant) [inline source/devices/AM62AX/linux/Overview/Download_and_Install_the_SDK.rst:98]: For all directives under this file, **Make sure that the note directive and the content block have 1 new line between them and that the content block has at least 1 newline after it.**

Refer CONTRIBUTING guidelines for more details - https://github.com/TexasInstruments/processor-sdk-doc/blob/master/CONTRIBUTING.md#indentation-and-whitespace

File: source/devices/AM62AX/linux/Overview/Download_and_Install_the_SDK.rst [hunk 2]
```diff
@@ -8,11 +8,11 @@ to start Linux development on the TI microprocessor.
...
 
-+--------------------------------------------+----------------------------------------------------------------------+
-| **Installer Details**                      | **Installers Name**                                                  |
-+--------------------------------------------+----------------------------------------------------------------------+
-| PSDK Linux AM62A Installer (Linux version) | ti-processor-sdk-linux-am62axx-evm-<version>-Linux-x86-Install.bin   |
-+--------------------------------------------+----------------------------------------------------------------------+
++--------------------------------------------+-----------------------------------------------------------------------------+
+| **Installer Details**                      | **Installers Name**                                                         |
++--------------------------------------------+-----------------------------------------------------------------------------+
+| PSDK Linux AM62A Installer (Linux version) | ti-processor-sdk-linux-edgeai-am62a-evm-<version>-Linux-x86-Install.bin     |
```
Comment (cshilwant) [inline source/devices/AM62AX/linux/Overview/Download_and_Install_the_SDK.rst:14]: Any reasons to not use "|\_\_SDK_INSTALL_FILE\_\_|" which you are already using in this file?

https://github.com/TexasInstruments/processor-sdk-doc/blob/master/configs/AM62AX/AM62AX_linux_config.txt#L22

File: source/devices/AM62AX/linux/Overview/Download_and_Install_the_SDK.rst
Comment (cshilwant) [inline source/devices/AM62AX/linux/Overview/Download_and_Install_the_SDK.rst:1]: The Package overview section in this file seems outdated. Misses references of `k3r5-devkit/`, `licenses/`, etc

**Review by cshilwant — CHANGES_REQUESTED** [review]

File: source/devices/AM62AX/linux/Overview/Download_and_Install_the_SDK.rst (hunk 2 above)
Comment (sinha-shreyash) [reply source/devices/AM62AX/linux/Overview/Download_and_Install_the_SDK.rst:14]: I was not aware of this, using |__SDK_INSTALL_FILE__| as a replacement variable under Installer Name

File: source/devices/AM62AX/linux/Overview/Download_and_Install_the_SDK.rst
Comment (sinha-shreyash) [reply source/devices/AM62AX/linux/Overview/Download_and_Install_the_SDK.rst:1]: Can you please share some reference for this

Comment (sinha-shreyash) [reply source/devices/AM62AX/linux/Overview/Download_and_Install_the_SDK.rst:1]: I have made some changes accordingly, please review and let me know

File: source/devices/AM62AX/linux/Overview/Download_and_Install_the_SDK.rst [hunk 3]
```diff
@@ -27,12 +27,14 @@ Note that this **does not** prevent the user from installing the SDK on other Li
 distributions.
 
 .. rubric:: Downloading the SDK installer
+
```
Comment (StaticRocket) [inline source/devices/AM62AX/linux/Overview/Download_and_Install_the_SDK.rst:30]: This is invalid syntax:
```
/build/source/devices/AM62AX/linux/Overview/Download_and_Install_the_SDK.rst:29: ERROR: Error in "rubric" directive:
no content permitted.

.. rubric:: Downloading the SDK installer

   :name: downloading-the-sdk-installer [docutils]
```

All of these rubrics are incorrect.

**Review by StaticRocket — CHANGES_REQUESTED** [review]

**Review by cshilwant — DISMISSED** [review]

**Review by StaticRocket — APPROVED** [review]

**Review by cshilwant — APPROVED** [review]

**Pull request merged**